draft-irtf-cfrg-hpke.md

```diff
@@ -510,9 +555,18 @@ The primary differences from the base case are:
 * The PSK ID is added to the context string used as the `info` input
   to the KDF
 
-This mechanism is not suitable for use with a low-entropy password
-as the PSK.  A malicious recipient that does not possess the PSK can
-use decryption of a plaintext as an oracle for performing offline
+The PSK SHOULD be of length Nh bytes or longer, and SHOULD have
+Nh bytes of entropy or more. Using a PSK shorter than Nh bytes
+is permitted. A PSK that is longer than Nh bytes or has more than
```

Comment:
Can we pull the text from #59 and place it here? (And move the next paragraph to the security considerations as is done in #59?)